src/components/GitHub.tsx

```diff
@@ -0,0 +1,61 @@
+import React, { useEffect, useState } from "react"
+import Admonition from "@theme/Admonition"
+
+type GitHubProps = {
+  json: any
+}
+
+export const PWithData: React.FunctionComponent<GitHubProps> = ({ json }) => {
+  return <div dangerouslySetInnerHTML={{ __html: json }} />
```

Comment:
Try taking raw HTML and render it into the page directly

Reply:
I have tried it and it's just not possible to make this work. It doesn't make it past the Docusaurus build if the HTML is injected directly into the MDX files.